jaeger/charts/linkerd-jaeger/templates/namespace-metadata-rbac.yaml

```diff
@@ -76,3 +77,4 @@ subjects:
 - kind: ServiceAccount
   name: namespace-metadata
   namespace: {{.Release.Namespace}}
+{{-end}}
```

Comment:
You require spaces here or else the template breaks, as you can see in the integration tests results.
```suggestion
{{- end }}
```

Reply:
Thanks  @alpeb . Added spaces between those end statements.

Reply:
Thanks, can you address my comment about naming above? After that, this PR looks fine from my perspective.